backend/src/test/java/com/wooteco/nolto/image/infrastructure/FfmpegConverterTest.java

```diff
@@ -0,0 +1,34 @@
+package com.wooteco.nolto.image.infrastructure;
+
+import com.wooteco.nolto.image.config.FfmpegConfig;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.net.URL;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = {FfmpegConverter.class, FfmpegConfig.class})
+class FfmpegConverterTest {
+
+    private final String gifFileName = "jjv1FK.gif";
+
+    @Autowired
+    private FfmpegConverter ffmpegConverter;
+
+    @Test
+    void convertGifToMp4() {
+        // given
+        URL resource = getClass().getClassLoader().getResource("static/" + gifFileName);
+        String gifFilePath = resource.getPath();
+        String expectedPath = "/Users/parksmac/IdeaProjects/2021-nolto/backend/build/resources/test/static/" + gifFileName;
+        assertThat(gifFilePath).isEqualTo(expectedPath);
+        int indexOfExtensionDot = gifFilePath.lastIndexOf(".");
+        String filePathWithoutExtension = gifFilePath.substring(0, indexOfExtensionDot);
+        String mp4FilePath = filePathWithoutExtension + ".mp4";
+
+        // when
+        ffmpegConverter.convertGifToMp4(gifFilePath, mp4FilePath);
```

Comment:
```suggestion
        assertDoesNotThrow(() -> ffmpegConverter.convertGifToMp4(gifFilePath, mp4FilePath));
```
문제 없이 정상 동작을 확인하는 테스트라면 이렇게는 어떠신가요??

Reply:
오 이거 몰라서 찾아보려다가 깜빡하고 안했네요